src/main.rs

```diff
@@ -67,11 +67,11 @@ async fn get_object(
         if let Ok(r) = range_header.to_str() {
             if let Some(bytes_range) = r.strip_prefix("bytes=") {
                 if let Some((start, end)) = bytes_range.split_once('-') {
-                    if let (Ok(s), Ok(e)) = (start.parse::<u64>(), end.parse::<u64>()) {
-                        range_start = s;
-                        range_end = e;
```

Comment:
You're no longer setting the `range_start` and `range_end` variables. These variables are used to construct the `Content-Range` header and as-is, this change would introduce a regression.